src/styles/blocks/stars

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display:flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &__star:nth-child(-n+4) {
+    background-image: url(/src/images/star-active.svg);
+  }
```

Comment:
make it work for all possible modifiers 
"all `stars--N` modifiers work as expected (Highlight first `N` stars)"